docs/source/UserGuide/VUDRM-Admin.md

```diff
@@ -0,0 +1,123 @@
+
+<img src="../_static/user-guide-images/4clientinfo.png" class="image" width="750" alt="VUALTO Admin - Client Info Page"/>
+
+If you require amendments to your account, please contact [VUALTO support](support@vualto.com).
+
+#### Client Info - Edit default token policies
+
+At the bottom of the `Client Info` page, you can view each DRM service’s default token policy. This can be edited to give more control over how a service will act by default.
+
+<img src="../_static/user-guide-images/5edittokenpolicies.png" class="image" width="750" alt="VUALTO Admin - Edit Token Policies"/>
+
+Clicking `Edit Token Policy` will load the JSON editor. 
+
+<img src="../_static/user-guide-images/5.1jsoneditor.png" class="image" width="750" alt="VUALTO Admin - Edit Token Policies - JSON Editor"/>
+
+Please ensure that the JSON has been entered correctly, otherwise you will be informed of an error and will not be able to save the amended policy.
+
+When the amends are complete, click `Close` and then `Save`. 
+
+Each DRM service has limitations to how their policy can be altered. Please read the links below for more details:
+
+* [DRM Policy](https://docs.vualto.com/projects/vudrm/en/latest/DeveloperDocumentation/VUDRM-token.html#drm-policy)
+* [PlayReady DRM Policy](https://docs.vualto.com/projects/vudrm/en/latest/DeveloperDocumentation/VUDRM-token.html#playready-drm-policy)
+* [Fairplay DRM Policy](https://docs.vualto.com/projects/vudrm/en/latest/DeveloperDocumentation/VUDRM-token.html#fairplay-drm-policy)
+* [Widevine DRM Policy](https://docs.vualto.com/projects/vudrm/en/latest/DeveloperDocumentation/VUDRM-token.html#widevine-drm-policy)
+
+### VUDRM Token
+
+In this section, you can create, validate, decrypt, encode and decode VUDRM Tokens.
+
+<img src="../_static/user-guide-images/6vudrmtoken.png" class="image" width="1000" alt="VUALTO Admin - VUDRM Token Page"/>
+
+There are a variety of templates to choose from which will automatically populate the Policy text box, you can then generate a token with those parameters. 
+
+The buttons on the page do the following:
+
+* `Generate`        - This will generate a token to include the policy you have entered
+* `Validate Policy` - This will ensure the policy is valid
+* `Validate Token`  - Validate the token created
+* `Decrypt Token`   - Decrypts the token
+* `Encode Token`    - URL Encodes the token
+* `Decode Token`    - URL Decodes the token
+
+For an in depth guide of how our tokens work - please refer to the [VUDRM Token documentation](https://docs.vualto.com/projects/vudrm/en/latest/DeveloperDocumentation/VUDRM-token.html#). 
+
+### VUDRM Encryption Keys
+
+Within this section, you can call the `CPIX Key Provider` to fetch VUDRM Encryption Keys in CPIX XML document format as well as the CPIX keys as JSON which is generated at the same time, both of which are available to copy. 
+
+We also have our `Legacy JSON Key provider` (available on the `Key Provider API` dropdown) to use if your product requires it.
```

Comment:
This should probably just be at the end of this section and say that although it is available we recommend using the CPIX key provider just so that new people are pushed more towards that instead.